Parent/Parent/Conversations/ComposeViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2019-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+
+class ComposeViewController: UIViewController, ErrorViewController {
+    weak var bodyMinHeight: NSLayoutConstraint!
```

Comment:
Why is this weak?

Reply:
It's not in IB, and the constraints are held strongly by views. I don't think there would be a loop if it was a strong reference, though so I can change it if you think that's better.

Reply:
Just curious about it.